.travis.yml

```diff
@@ -1,18 +1,15 @@
 dist: xenial
 
 language: python
-python: 3.6
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
 
-install: pip install tox-travis codecov==2.0.5
-script: tox
-after_success: codecov
-deploy:
-  provider: pypi
-  user: sallyruthstruik
-  password:
-    secure: AO14vBQPpz2UiVZKETQk4D3zjoDzjJjXIRbohnSiQrtPJ9yafF/diLa606McyhZx3mRug3MpJY9dWCTq+0x5ne3NExkOcHC5kz30lIUIK1D2xlHaSteVtT/4hFf0Rz09huTsgOczDzBKww7nDvNCVQ94MLrEs0H0RXt7HZpnVsWZcYDhJovJ4l+3nhas+jjQrhhRUx9ePVvQGjlIJ96UcXYZISlFahodCiLPmoZFyBvdVSwxcJbwnUpsM8NLKnCXy8mzcDRsQCOCS9hsX/H+erErzBoVyrWibJ6KtuZHBcoXLDMV4IBcQgXAUz2RXpSjR3/nb96fAd1tHa5hel6OVyDVsJEQHm682+0Rn89LjXww8eskJEa1TZnOy+r0+zBYKquUo3A0EkYvFIAd/Ucmf3u+KWDlHOGKKXTc2Iu112M78Ih7JgIvw/b0CS7vGd9wOPD6kjLbmZe9JQH0DuCOo5K/J47+ikJnyaAPtJw4MeOggXFZq6QVZIda8vCR1VBP/6lXuBBeF0rZDa4ck+WCkIFIpoXiwYxuQripE9zdUSJsDlb+GTV2XhK7rUIS9u1RhmNSfSV9K+uOpBfOwhTtjJDH96FCQqtiwb+9xLSkuG+5cJnd6slDoVniAGiN9MO/nrJaghSUTbB9GeY1503Tot/xl3lYXUFc3+8rQIQVH0E=
-  on:
-    tags: true
```

Comment:
As per our email, you may want to hang on to this bit.

Reply:
Thanks, rolled back in e23f5fbf730e7464df852f5d21121e4b453fd07c.